WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -2008,14 +2020,6 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
         case WPAnalyticsStatPerformedCoreDataMigrationFixFor45:
         case WPAnalyticsStatMaxValue:
             return nil;
-
-        // The following are implemented as part of the release/14.3 branch
-        // https://github.com/wordpress-mobile/WordPress-iOS/pull/13505
-        // For the time being I'm stubbing these out until that branch makes it back to develop
-        case WPAnalyticsStatWelcomeNoSitesInterstitialShown:
-        case WPAnalyticsStatWelcomeNoSitesInterstitialButtonTapped:
-        case WPAnalyticsStatWelcomeNoSitesInterstitialDismissed:
-            return nil;
```

Comment:
Unittests were failing to compile due to duplicate cases so I removed these considering the code comment above. @emilylaguna let me know if this looks OK. Thanks!

Reply:
@pinarol the file looks good to me. I also confirmed the unit tests are building and running correctly for me as well.